neural_search/operations/default.json

```diff
@@ -18,12 +18,24 @@
     }
   }
 },
+{
+  "name": "put-concurrent-segment-search-setting",
```

Comment:
can we call this conditionally, only in case user set some value? With latest commit by default we disable CSS if user hasn't provide any value. Problem is - starting from 3.0 CSS is "enabled" by default, and in 2.19 and below default state is disabled. Because we don't know the version of the OS we need to allow system itself to set the default value.